iica-catie.csl

```diff
@@ -59,24 +58,22 @@
   </macro>
   <macro name="locators">
     <choose>
-      <if type="article-journal article-magazine article-newspaper" match="any">
+      <if type="article-magazine article-newspaper article-journal" match="any">
         <group delimiter=", ">
           <group>
             <choose>
               <if variable="volume">
-                <text macro="volume" prefix="Año " suffix=", "/>
+                <text macro="volume" prefix=" "/>
               </if>
-              <else>
-                <text value="s.d. "/>
-              </else>
             </choose>
-            <text variable="issue" prefix="N° "/>
+            <text variable="issue" prefix="(" suffix=")"/>
+            <text variable="page" form="short" strip-periods="false" font-variant="normal" display="block" prefix=":"/>
```

Comment:
`strip-periods="false" font-variant="normal" display="block"` should be removed here.

Reply:
ok